packages/eslint-plugin/configs/recommended.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 		require.resolve( './custom.js' ),
 		'plugin:@wordpress/eslint-plugin/recommended',
 	],
+	parser: '@typescript-eslint/parser',
```

Comment:
Added the typescript-eslint parser as was suggested [here](https://github.com/gajus/eslint-plugin-jsdoc/issues/604#issuecomment-653962767)
To fix this import error:
`Syntax error in type: import('@wordpress/element').WPElement} WPElement  jsdoc/valid-types`
An import statement like this was done in more place (you can see the errors in the initial travis ci output).